auth-api/src/auth_api/services/affiliation.py

```diff
@@ -170,11 +171,13 @@ def create_affiliation(org_id, business_identifier, pass_code=None, certified_by
 
         current_app.logger.debug('<create_affiliation find affiliation')
         # Ensure this affiliation does not already exist
-        affiliation = AffiliationModel.find_affiliation_by_org_and_entity_ids(org_id, entity_id)
+        affiliation = AffiliationModel.find_affiliation_by_org_and_entity_ids(org_id, entity_id, environment)
         if affiliation is not None:
             raise BusinessException(Error.DATA_ALREADY_EXISTS, None)
 
-        affiliation = AffiliationModel(org_id=org_id, entity_id=entity_id, certified_by_name=certified_by_name)
+        affiliation = AffiliationModel(org_id=org_id, entity_id=entity_id, certified_by_name=certified_by_name,
+                                       environment=environment)
+        affiliation.environment = environment
```

Comment:
Is this needed since it is already being passed in when instantiating the model?